### ISSUE TITLE
Molecular weight

### ISSUE THREAD

**Issue by jowr** [ibell/coolprop#187]
The SI units for molecular weight should be kg/mol and not g/mol...

**Comment by ibell** [comment]
Correct.  Can you fix that in the code? And in the docs in CoolProp.pyx?

**Comment by jowr** [comment]
Yes, I already added it to the conversion functions...

**Comment by jowr** [comment]
Puh, it is used so many places... I think I won't change it any time soon.

**Comment by ibell** [comment]
I guess the point is that as long as the conversion function is fixed
we are good for 4.2, and in v5, everything will be SI.  No sense
putting more work into the code in 4.2.  We should change as little as
possible in 4.2.

By the way, are we ready to release 4.2?  Can you give me a hand with
the release notes?

**Comment by jowr** [comment]
Yes, no problem.

But can we wait until I finished ExternalMedia?

I currently integrate CoolProp and ExternalMedia and the C++ part is
almost done...

**Comment by ibell** [comment]
Sure that's fine by me - theres nothing really pressing in 4.2 except
for some problems with conductivty of R134a.  I can put some work into
the release notes while we wait.

**Comment by jowr** [comment]
I moved the CPP files for the Modelica wrapper to the ExternalMedia SVN. I guess future changes in ExternalMedia are independent from CoolProp. Hence, we can release 4.2. I'll have a look at the release notes and see if I can add something.

**Comment by ibell** [comment]
Seems PropsSI still returns mole mass in kg/kmol, I guess either we
fix the docs or we fix the output...  For 4.2 either is fine by me.
